extensions/amp-story-shopping/0.1/test/test-amp-story-shopping-attachment.js

```diff
@@ -44,7 +47,8 @@ describes.realWin(
       const attachmentChildEl = shoppingEl.querySelector(
         'amp-story-page-attachment'
       );
-      const attachmentChildImpl = await attachmentChildEl.getImpl();
+      const attachmentChildImpl = new AmpStoryPageAttachment(attachmentChildEl);
```

Comment:
We should really be able to keep the previous implementation here, this is hacky